client/tmclient/screens.py

```diff
@@ -266,11 +271,23 @@ def handle_game_input(self, text):
             quit_client(self)
         elif text.startswith('/edit'):
             #TODO check for active witch editor
-            text = text[1:]
+            obj_name = text.split("/edit ")[1]
+            if not self.valid_edit_object(obj_name):
```

Comment:
Proposing we don't do any client side validation and let the server determine if somethings parses to a valid in scope object.

Reply:
i initially made the scope list because i wanted to eventually have the witch panel show a list of objects in scope when nothing's loaded for editing? though it's probably a good idea to have the server do the validation (which iirc it already does)